index.html

```diff
@@ -5,8 +5,39 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Alex Portfolio</title>
+    
+    <!--Style-->
+    <link rel="stylesheet" href="./style/style.css"> 
+
+    <!--Roboto-->
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <!--Crete round-->
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <!--Crete round-->
+    <link href='https://fonts.googleapis.com/css2?family=Poppins' rel='stylesheet'>
+
 </head>
 <body>
-    
+    <header>
+
+        <nav>
+            <ul>
+                <li class="my-logo">Alex Dev</li>
+                <li>
+                    <a href="#">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="white" class="bi bi-list" viewBox="0 0 16 16">
+                            <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+                        </svg>
+                    </a>
+                </li>
+            </ul> 
+        </nav>
+
+    </header>
+    <section>
+        <h1>Hey There. I'm Alex</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>I can help you build a product, feature or website Look trough some of my work and experience! If you like what you see and have a project you need coded, don't heastiate contact me.</p>
+    </section>
```

Comment:
- [ ] As you can see in the original design. The section has a list of icons. Kindly add the missing icons. Following the original design is a must for this project.
![image](https://user-images.githubusercontent.com/43698511/129241988-a27b2853-c140-48b8-97c2-d7c1c3439801.png)

- [ ] I think your version of the project looks a bit different from the original design in terms of margins. Kindly use the same margins of the original design.
![image](https://user-images.githubusercontent.com/43698511/129242405-bd184f18-b9e7-4f4a-97c6-0453e9523015.png)